src/clients/evm/abis/SpaceFactory.json

```diff
@@ -0,0 +1,232 @@
+[
```

Comment:
Ideally this should use the compact format, you can get it with this function https://github.com/snapshot-labs/snapshot.js/blob/master/test/abi.ts#L22